src/extension/intents/node/agentIntent.ts

```diff
@@ -76,7 +78,8 @@ export const getAgentTools = (instaService: IInstantiationService, request: vsco
 			allowTools[ToolName.ReplaceString] = await modelSupportsReplaceString(model);
 			allowTools[ToolName.ApplyPatch] = await modelSupportsApplyPatch(model) && !!toolsService.getTool(ToolName.ApplyPatch);
 
-			if (allowTools[ToolName.ApplyPatch] && modelCanUseApplyPatchExclusively(model) && configurationService.getExperimentBasedConfig(ConfigKey.Internal.Gpt5ApplyPatchExclusively, experimentationService)) {
+			const applyPatchExclusively = configurationService.getExperimentBasedConfig(ConfigKey.Internal.Gpt5ApplyPatchExclusively, experimentationService) || simTestContext.isInSimulationTests;
```

Comment:
You can just nuke `Gpt5ApplyPatchExclusively`, it's at 100% on Insiders already and I was going to remove it anyway

Reply:
Oh nice, thanks